src/search.zig

```diff
@@ -201,32 +205,37 @@ fn matchPackageAux(packagesDir: std.fs.Dir, query: mvzr.Regex, manifestName: []c
 
     // does the package name match?
     if (query.isMatch(lowerStem)) {
-        try state.matches.append(try SearchMatch.init(allocator, stem, version, null));
+        try state.matches.append(try SearchMatch.init(allocator, stem, version, std.ArrayList([]const u8).init(allocator)));
```

Comment:
In practice `init` does not allocate, but this just happens to be the case. For good measure I would still add a defer `deinit`. The easiest would be to just move L210-211 above the `if`